frontend/ui/lib/widgets/place_block_widget.dart

```diff
@@ -13,17 +13,40 @@ class _PlaceBlockWidgetState extends State<PlaceBlockWidget> {
   final String placeName;
   _PlaceBlockWidgetState(this.placeName);
 
+  List<bool> _selections = List.generate(3, (_) => false);
+
   @override
   Widget build(BuildContext context) {
-    return Container(
-      decoration: BoxDecoration(
-          border: Border.all(
-            color: Colors.black,
-          ),
-          borderRadius: BorderRadius.all(Radius.circular(5))),
-      margin: const EdgeInsets.all(15.0),
-      padding: const EdgeInsets.all(10.0),
-      child: Text(placeName),
-    );
+    return ExpansionTile(
+        title: Text(placeName),
+        trailing: ToggleButtons(
+          children: <Widget>[
+            Text('M'),
+            Text('T'),
+            Text('D'),
```

Comment:
We probably don't want a "D" (delete) toggle button here, but a separate button that actually allows the trip to be removed. (Unless I misunderstand the experience we want on this view.)

Reply:
I think we still want a button to delete a place visit but I agree that it should be separate from the M & T button (like maybe on the left side or whatever)